### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by joseantonioabellan84** [aclap-dev/video-downloadhelper#531] [Help and Q&A]
### Your question

Download failed

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://wpoptimizers.xyz/leccion/intro-optimizar-hosting/",
  "error": {
    "details": "SideDownload error: : [hls @ 0x11f605960] Error when loading first segment 'https://vz-1cdc1030-883.b-cdn.net/31788749-1be3-4986-a60e-4c9b8328fcf8/842x480/video0.ts?v=0&resolution=842x480&server=1365&contextId=1fad5f33-7e98-4915-b802-68dbb2d2c296'\n[in#0 @ 0x11f605850] Error opening input: Invalid data found when processing input\nError opening input file https://iframe.mediadelivery.net/31788749-1be3-4986-a60e-4c9b8328fcf8/842x480/video.drm?contextId=1fad5f33-7e98-4915-b802-68dbb2d2c296.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_7189612650864421",
    "error": "coapp_failure",
    "id": "coapp_failure_ce3d4e40-3dcd-4af1-8cb1-0b340dbc2569"
  }
}

**Comment by GrampaWildWilly** [comment]
Refer to #530.
